src/components/AdvisorList/index.js

```diff
@@ -33,19 +33,15 @@ export default class Advisors extends Component {
           ? (advisorText += `${advisor.Firstname + ' ' + advisor.Lastname}`)
           : (advisorText += `${advisor.Firstname + ' ' + advisor.Lastname}, `);
       });
-      content = <Typography>{advisorText}</Typography>;
+      content = <Typography style={this.props.styles}>{advisorText}</Typography>;
 
       if (this.props.advisors.length === 0) {
         advisorPrefix = (
           <div>
             <Typography>Advisor:</Typography>
           </div>
         );
-        content = (
-          <div>
-            <Typography>(not assigned)</Typography>
-          </div>
-        );
+        content = <Typography style={this.props.styles}>None assigned</Typography>;
```

Comment:
Does removing the div here cause any CSS issues? I know sometimes empty divs can have unexpected results, so I wasn't sure if this one was important for spacing of something like that.